src/components/MaterialUI/DataDisplay/Modal.js

```diff
@@ -85,13 +85,13 @@ const Modal = ({ message, modalProps }) => {
 	const backdropClickCallback = modalProps?.get(ModalProps.propNames.backdropClickCallback);
 	const modalType = modalProps?.get(ModalProps.propNames.type) ?? "normal";
 
-	var containerCls = clsx({
+	const containerCls = clsx({
```

Comment:
@joclefrancois maybe replace clsx to classNames here while you're there. I was going to do that later